src/plccng/Scanner/matcher/matcher_test.py

```diff
@@ -0,0 +1,33 @@
+from ...load_spec.parse_spec import parse_lexical_spec
+from ...load_spec.structs import Line
+
+from plccng.Scanner.matcher import matcher
+
+def test_rules_attribute():
```

Comment:
I don't think clients not need access to a rules attribute. Wether or not a Matcher has such an attribute is an internal detail that will be tested by other tests.

So we don't want this test.